draft-ietf-rats-corim.md

```diff
@@ -1491,36 +1491,97 @@ and how the verifier matches a Reference Value against the Accepted Claims Set
+
+#### Matching a Conditional Endorsement Series Triple against the Accepted Claims Set
+
+A Conditional Endorsement Series Triple {{sec-comid-triple-cond-series}} is an efficient
+encoding of a series of alternative Reference Values, each with a corresponding
+Endorsed Value. The Verifier adds the Endorsed Values corresponding to the first
+Reference Value which matches.
+
+All Reference Values and Endorsed Values in the triple use the same `environment-map`
+which is encoded within the `stateful-environment-record` part of the triple.
+
+Reference Values are encoded in two parts of this triple.
+
+Reference Values which apply to every Endorsed Value are encoded in the
+`stateful-environment-record`, in the same way as for the
+Conditional Endorsement Triple. If any of these Reference Values
+do not match Evidence or Endorsements in the Accepted Claims Set then then the
+entire triple is silently ignored.
+
+Reference Values which apply to only one Endorsed Value are encoded, with their
+matching Endorsed Value, in an array of `conditional-series-record`s.
+Each `conditional-series-record` encodes the `measurement-value-map` part for its
+Reference Values and for the corresponding Endorsed Value.
+
+The entries in the `conditional-series-record` array are processed in order. The verifier
+SHALL attempt to match the Reference Value part of each entry against entries
+in the Accepted Claims Set (see {{sec-match-one-ref-val}}).
+If the Reference Value matches then the verifier SHALL add the corresponding
+Endorsed Value to the Accepted Claims Set (see {{sec-add-to-acs}}) and
+SHALL skip processing the remaining entries in the array.
 
-Each group is processed independently of other groups. If a group fails to match
-the Accepted Claims Set then this does not affect the processing of other groups.
+>> *TODO: Is a Conditional Endorsement Series Triple affected by
+`reference-triple-record`s in the same group? I think not.
+Does this need to be mentioned?*
 
 #### Matching a Reference Value against the Accepted Claims Set {#sec-match-one-ref-val}
 
-This section describes how a Reference Value is matched against Evidence in the Accepted
-Claims Set.
-If any part of the processing indicates that the Reference Value does not match then the remaining steps in this section are skipped for that group.
+This section describes how a Reference Value is matched against Evidence or Endorsements
+in the Accepted Claims Set.
+If any part of the processing indicates that the Reference Value does not match
+then the remaining steps in this section are skipped for that group.
 
-A Reference Value consists of an `environment-map` plus a `measurement-map`. In the
-`reference-values-triple-record` these are packaged together. In other triples multiple
-Reference Values are represented more compactly by letting one `environment-map`
-apply to multiple `measurement-map`s.
+A Reference Value consists of an `environment-map` plus a `measurement-map`. It is
```

Comment:
I still think this is too narrowly stated. If taken literally, the identity-triples, attest-key-triples, conditional triples and coswid-triples have to be discarded because their "measurements" aren't describes as `measurement-map`.

Reply:
As you see below I think the spec needs a name for `environment-map` plus `measurement-map` when it appears in conditional triples as well as when it appears in reference-values-triple-record.
I am quite happy to use a new name for EM+MM.
What should it be?

Reply:
Possible suggestions:
environment-measurement-pair
claim tuple

Reply:
> environment-measurement-pair

Fine! but probably would not use hyphens with pair. E.g.: "... an environment-measurement pair..." or "... an environment-measurement pairing..."